tutoraspects/patches/local-docker-compose-dev-services

```diff
@@ -2,32 +2,23 @@
 superset:
   {% include 'base-docker-compose-services' %}
     OPENEDX_LMS_ROOT_URL: "http://{{ LMS_HOST }}:8000"
+    OAUTH2_CLIENT_ID: {{ SUPERSET_OAUTH2_CLIENT_ID_DEV }}
   command: ["bash", "/app/docker/docker-bootstrap.sh", "app-gunicorn"]
   ports:
     - 8088:{{ SUPERSET_PORT }}
   depends_on:
     - mysql
     - redis
     - superset-worker
-    - superset-worker-beat
```

Comment:
How come the beat container is being removed? Seems like a thing we might want for alerts and emailing reports on a schedule?

Reply:
This beat service was just another worker. The beat command should be 
`"bash", "/app/docker/docker-bootstrap.sh", "beat"`

Do we need this for local development if we already have worker and the beat is just eating resources?

Reply:
Forget it, I see that this is needed for alerts

Reply:
@bmtcril I did found out that this service was not running in k8s, so i also added support for it in k8s